pkg/controller/expcertificates/issuing/issuing_controller_test.go

```diff
@@ -450,6 +456,332 @@ func TestIssuingController(t *testing.T) {
 			},
 			expectedErr: false,
 		},
+
+		"if certificate is in Issuing state with temp annotation, one CertificateRequest Pending, no target Secret, create target secret with temporary certificate": {
+			certificate: exampleBundle.Certificate,
+			builder: &testpkg.Builder{
+				CertManagerObjects: []runtime.Object{
+					gen.CertificateFrom(issuingCert,
+						gen.AddCertificateAnnotations(map[string]string{
+							cmapi.IssueTemporaryCertificateAnnotation: "true"},
+						),
+					),
+					gen.CertificateRequestFrom(exampleBundle.CertificateRequestPending,
+						gen.AddCertificateRequestAnnotations(map[string]string{
+							cmapi.CertificateRequestRevisionAnnotationKey: "2",
```

Comment:
What is `2` relative to the existing `status.revision` on the Certificate?

Reply:
The current revision on the cert is 1

Reply:
👍 can you make that a comment?